components/layout/breakpointSplit/src/index.js

```diff
@@ -47,7 +47,7 @@ class LayoutBreakpointSplit extends Component {
         )}
       />
     )))
-    return (<span className='sui-LayoutBreakpointSplit'>{content}</span>)
+    return (<span className={cx(this.props.className, 'sui-LayoutBreakpointSplit')}>{content}</span>)
```

Comment:
i'm more used to see something like:

```js
const className = cx(
     'sui-LayoutBreakpointSplit',
     this.props.className
  )
return (
  <span className={className}>{content}</span>
)
```

what do you think? to me looks a bit more clear.

Reply:
i'm agree! done!